modules/check_in/app/serializers/check_in/v2/appointment_data_serializer.rb

```diff
@@ -93,7 +54,20 @@ class AppointmentDataSerializer
           { appointments: appointments }
         end
       end
-      # rubocop:enable Metrics/BlockLength
+
+      def self.address_helper(address)
+        {
+          street1: address[:street1],
+          street2: address[:street2],
+          street3: address[:street3],
+          city: address[:city],
+          county: address[:county],
+          state: address[:state],
+          zip: address[:zip],
+          zip4: address[:zip4],
```

Comment:
Could you please confirm if the payload from downstream sends `mailing_address` and `home_address` with `nil` in `zip4`?  If there is no `zip4` key in the payload, this change might break the existing use-case.

Reply:
downstream (LoROTA) in this case doesn't send zip4 in some addresses (demographics). In that case, this would be nil and we'll pass it to the front-end with `"zip4": null,` in json.

Reply:
This is correct. LGTM!